Assets/Mapzen/VectorData/Filters/PropertyValueFeatureMatcher.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace Mapzen.VectorData.Filters
+{
+    public class PropertyValueFeatureMatcher : PropertyFeatureMatcher
+    {
+        public List<object> ValueSet { get; set; }
+
+        protected override bool MatchesProperty(object property)
+        {
+            return ValueSet.Contains(property);
```

Comment:
Using direct comparison here without type casting can lead to some issues, here is an example:

Trying to filter out all features that have a `height` value of `417`:
```c#
var buildingExtrusionFilter = new FeatureFilter().TakeAllFromCollections("buildings").Where(FeatureMatcher.HasPropertyWithValue("height", 417));
```

```
> ValueSet
Count=1
	[1]: 417
	Raw View: 
> ValueSet.Contains(property)
false
> property == 417
false
> (int)property
417
> (int)property == 417
true
```

A possibility is to type check using introspection first, if they can be represented as the same type, typecast to the type of the `ValueSet`. only then we can safely send it to the helper method `Contains`.